www/js/da.js

```diff
@@ -160,7 +160,7 @@ function formatResults(data) {
 }
 
 function formatEntry(data) {
-    var template = doT.template("<h3>{{=it.term}}</h3><div>{{=it.def}}</div>",
+    var template = doT.template("<h3>{{=it.term}}</h3><div>{{=it.def}}</div><i>{{=it.def}}</i>",
```

Comment:
nao queres o it.def, queres mesmo a tag direitinha como me tinhas mostrado no face. `<i class="..."></i>`. Não tem que inventar. Cuidado só com as aspas.